src/components/Permalink/Permalink.js

```diff
@@ -36,6 +36,10 @@ const LINE_FILTER = 'PublishedLineName';
 const ROUTE_FILTER = 'TripNumber';
 const OPERATOR_FILTER = 'Operator';
 
+const getUrlParam = param => {
+  return param ? decodeURI(param).replace(/\s+/g, '') : undefined;
```

Comment:
```suggestion
  return decodeURI(param).replace(/\s+/g, '');
```

Reply:
In fatc there is a decode option for qs.parse .https://www.npmjs.com/package/query-string#decode